mlflow/protos/databricks_service.proto

```diff
@@ -0,0 +1,182 @@
+// This file contains the proto definition for communicating with the Databricks tracking server.
+// The message definition should be kept in (mostly) sync with the MLflow service definition.
+syntax = "proto2";
+
+package mlflow.databricks;
+
+import "assessments.proto";
+import "spans.proto";
+import "databricks.proto";
+import "scalapb/scalapb.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option py_generic_services = true;
+
+service DatabricksService {
```

Comment:
@serena-ruan We already have other Databricks-related proto files. Can we name this more specifically?

Reply:
What about DatabricksTrackingService?

Reply:
This stuff is specific to tracing, right?

Reply:
Yes that's fine, we may want to remove `databricks_trace_server.proto` later to reduce confusion